File: Java/BaekJoon/Bronze/B4_5524.java
package BaekJoon.Bronze;

import java.util.Scanner;

public class B4_5524 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            String name = sc.next();

            System.out.println(name.toLowerCase());
        }

        sc.close();
    }
}

File: C/BaekJoon/Bronze/B4_5524.c
#include <stdio.h>

int main() {
  int n;
  scanf("%d", &n);

  for (int i = 0; i < n; i++) {
    char name[30];
    scanf("%s", name);

    for (int j = 0; name[j] != '\0'; j++) {
      if (name[j] >= 'A' && name[j] <= 'Z') {
        printf("%c", name[j] + 32);
      } else {
        printf("%c", name[j]);
      }
    }

    printf("\n");
  }

  return 0;
}